src/game.cpp

```diff
@@ -7899,12 +7902,15 @@ bool game::npc_menu( npc &who )
     amenu.addentry( cancel, true, 'q', _("Cancel") );
 
     const bool obeys = debug_mode || (who.is_friend() && !who.in_sleep_state());
+    const bool armed = who.is_armed();
     amenu.addentry( swap_pos, obeys, 's', _("Swap positions") );
     amenu.addentry( push, obeys, 'p', _("Push away") );
     amenu.addentry( examine_wounds, true, 'w', _("Examine wounds") );
     amenu.addentry( use_item, true, 'i', _("Use item on") );
     amenu.addentry( sort_armor, true, 'r', _("Sort armor") );
     amenu.addentry( attack, true, 'a', _("Attack") );
+    amenu.addentry( disarm, armed, 'd', _("Disarm") );
+    amenu.addentry( steal, true, 'S', _("Steal") );
```

Comment:
Stealing and disarming friendly NPCs doesn't really make sense since they give up their items voluntarily. In those cases, "steal" could be changed to "take" and cause no hostility.